.changeset/tiny-shrimps-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Accordion` - added `@size`, `@type`, and `@isStatic` arguments
```

Comment:
In Figma we updated the @ type argument to containerType

Reply:
Thanks for the heads-up – I saw your comment in the CRD yesterday. In the Ember version of this component, we don't have the concept of a container (it could be interpreted as the container that holds the revealed content or something else). Can you provide some background on what prompted this change (in the CRD or here)?

Reply:
@alex-ju @majedelass I want to update this comment as resolved via Slack thread [here](https://hashicorp.slack.com/archives/C025N5V4PFZ/p1718715573875349) 
---
We've decided to move from containerType in Figma and align it with the code. So now both Code and Figma are Type.